lib/install/management/upgrade.go

```diff
@@ -85,7 +85,9 @@ func (d *Dispatcher) Upgrade(vch *vm.VirtualMachine, conf *config.VirtualContain
 	}
 
 	if err = d.update(conf, settings); err == nil {
-		d.retryDeleteSnapshot(snapshotName, conf.Name)
+		// TODO don't save too many old snapshots.. uncomment this and delete old ones
```

Comment:
yes that was my original intent / design and I forgot to actually add it in :) I will go ahead and add in that functionality.